### ISSUE TITLE
Fixing/Buffing Villager recovery 

### ISSUE THREAD

**Issue by Snakkattakk** [HDR-Development/HewDraw-Remix#914]
TLDR: Give villager Balloon Trip Cancel (BTC) but not Quick Balloon Trip Cancel (QBTC)

So back when Isabelle got the up special cancels added it was talked about and decided villager shouldn't get the cancel since he has lloid mine to help cover himself and recover already. I'm just going to advocate for someone taking a second look at this and seeing if he should get the cancel as well. 

Personally I think he should get a cancel, but maybe not the early cancel window since he has the drag down back air and oppressive projectiles. This way it stays more of a recovery tool than a mixup tool like it is on Isabelle.
